odoo/models.py

```diff
@@ -3150,6 +3150,18 @@ def get_base_url(self):
         self.ensure_one()
         return self.env['ir.config_parameter'].sudo().get_param('web.base.url')
 
+    def is_child_of(self, others):
```

Comment:
```suggestion
    def is_child_of(self, others, func=any):
```

and replace `any` by `func` in the body